packages/rangelink-vscode-extension/scripts/verify-qa-scripts.sh

```
@@ -0,0 +1,53 @@
+# Generated files are kept — they're valid artifacts you may want to inspect.
+#
+# Usage: ./scripts/verify-qa-scripts.sh
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NC='\033[0m'
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+PACKAGE_DIR="$(dirname "$SCRIPT_DIR")"
+
+cd "$PACKAGE_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)